src/components/Team/ActiveObjectives/wrapper.tsx

```diff
@@ -33,9 +33,17 @@ export interface GetTeamActiveObjectivesQuery {
   }
 }
 
+export const isUpdatedNeededOnObjectivesState = atom({
```

Comment:
IMHO you should keep this atom at `src/state/recoil/objective`

Reply:
fixed in 9844644